src/psyclone/tests/psyir/transformations/transformations_test.py

```diff
@@ -355,77 +396,6 @@ def test_omplooptrans_properties():
             in str(err.value))
 
 
-def test_parallellooptrans_validate_dependencies(fortran_reader):
-    ''' Test that the parallellooptrans validation checks for loop carried
-    dependencies. '''
-
-    def create_loops(body):
-        psyir = fortran_reader.psyir_from_source(f'''
-        subroutine my_subroutine()
-            integer :: ji, jj, jk, jpkm1, jpjm1, jpim1
-            real, dimension(10, 10, 10) :: zwt, zwd, zwi, zws
-            real :: total
-            {body}
-        end subroutine''')
-        return psyir.walk(Loop)
-
-    # Use OMPLoopTrans as a concrete class of ParallelLoopTrans
-    omplooptrans = OMPLoopTrans()
-    # Example with a loop carried dependency in jk dimension
-    loops = create_loops('''
-        do jk = 2, jpkm1, 1
-          do jj = 2, jpjm1, 1
-            do ji = 2, jpim1, 1
-              zwt(ji,jj,jk) = zwd(ji,jj,jk) - zwi(ji,jj,jk) * &
-                              zws(ji,jj,jk - 1) / zwt(ji,jj,jk - 1)
-            enddo
-          enddo
-        enddo''')
-
-    # Check that the loop can not be parallelised due to the loop-carried
-    # dependency.
-    with pytest.raises(TransformationError) as err:
-        omplooptrans.validate(loops[0])
-    assert ("Transformation Error: Dependency analysis failed with the "
-            "following messages:\nError: The write access to 'zwt(ji,jj,jk)' "
-            "and the read access to 'zwt(ji,jj,jk - 1)' are dependent and "
-            "cannot be parallelised. Variable: 'zwt'." in str(err.value))
-
-    # However, the inner loop can be parallelised because the dependency is
-    # just with 'jk' and it is not modified in the inner loops
-    omplooptrans.validate(loops[1])
-
-    # Reductions also indicate a data dependency that needs to be handled, so
-    # we don't permit the parallelisation of the loop (until we support
-    # reduction clauses)
-    loops = create_loops('''
-        do jk = 2, jpkm1, 1
-          do jj = 2, jpjm1, 1
-            do ji = 2, jpim1, 1
-              total = total + zwt(ji,jj,jk)
-            enddo
-          enddo
-        enddo''')
-    with pytest.raises(TransformationError) as err:
-        omplooptrans.validate(loops[0])
-    assert ("Transformation Error: Dependency analysis failed with the "
```

Comment:
Presumably, this functionality of the DA is tested as part of the DA suite?

Reply:
It is, it's one of the test cases in "test_scalar_parallelise"